src/lib/overrides.ts

```diff
@@ -56,18 +56,18 @@ const overrides: IOverrides = {
   global: {
     typings: [
       'declare const SERVER: boolean',
-      'declare const CLEINT: boolean',
+      'declare const CLIENT: boolean',
       'declare const MENU: boolean',
       'declare const GAMEMODE: Gamemode',
       'declare const GM: Gamemode',
 
-      'declare const SEWP: _SEWP',
+      'declare const SWEP: SWEP',
```

Comment:
Shouldn't this be `declare const SWEP: _SWEP`?

Reply:
Should be, not sure how I ended up overlooking this.
Will close the PR until I can get my hands back on it.

Reply:
I mean you can leave the PR open and just continue to commit till its good ;P